examples/model_compress/quantization/mixed_precision_speedup_cifar10.py

```diff
@@ -0,0 +1,319 @@
+        test_loss, 100 * correct / len(test_loader.dataset)))
+
+def check_accuracy(preds, labels):
+    start_idx = 0
+    num_correct_all = 0
+    for pred in preds:
+        pred = np.argmax(pred.reshape(-1, 10), axis=1)
+        effective_shape = pred.shape[0]
+        num_correct = np.count_nonzero(np.equal(pred, labels[start_idx:start_idx+effective_shape]))
+        num_correct_all = num_correct_all + num_correct
+        start_idx = start_idx + effective_shape
+    print("accuracy: ", 100 * num_correct_all / 10000)
+
+def main():
+    torch.manual_seed(0)
+    device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
+    train_loader = torch.utils.data.DataLoader(
+        datasets.CIFAR10('./data.cifar10', train=True, download=True,
+                         transform=transforms.Compose([
+                             transforms.ToTensor(),
+                             transforms.Normalize((0.4914, 0.4822, 0.4465), (0.2023, 0.1994, 0.2010))
+                         ])),
+        batch_size=64, shuffle=True)
+    test_loader = torch.utils.data.DataLoader(
+        datasets.CIFAR10('./data.cifar10', train=False, transform=transforms.Compose([
+            transforms.ToTensor(),
+            transforms.Normalize((0.4914, 0.4822, 0.4465), (0.2023, 0.1994, 0.2010))
+        ])),
+        batch_size=200, shuffle=False)
+    model = resnet18()
+    model.to(device)
+    train_epoch = 1
+    optimizer = torch.optim.SGD(model.parameters(), lr=0.1, momentum=0.9, weight_decay=5e-4)
+    scheduler = MultiStepLR(
+        optimizer, milestones=[int(train_epoch*0.5), int(train_epoch*0.75)], gamma=0.1)
+        
+    for epoch in range(train_epoch):
+        train(model, device, train_loader, optimizer, epoch)
+        scheduler.step()
+        acc = test(model, device, test_loader)
+
+    config = {
+        'conv1':{'weight_bit':8, 'activation_bit':8},
+        'layer1.0.conv1':{'weight_bit':8, 'activation_bit':8},
+        'layer1.0.conv2':{'weight_bit':8, 'activation_bit':8},
+        'layer1.1.conv1':{'weight_bit':8, 'activation_bit':8},
+        'layer1.1.conv2':{'weight_bit':8, 'activation_bit':8},
+        'layer2.0.conv1':{'weight_bit':8, 'activation_bit':8},
+        'layer2.0.conv2':{'weight_bit':8, 'activation_bit':8},
+        'layer2.1.conv1':{'weight_bit':8, 'activation_bit':8},
+        'layer2.1.conv2':{'weight_bit':8, 'activation_bit':8},
+        'layer3.0.conv1':{'weight_bit':8, 'activation_bit':8},
+        'layer3.0.conv2':{'weight_bit':8, 'activation_bit':8},
+        'layer3.1.conv1':{'weight_bit':8, 'activation_bit':8},
+        'layer3.1.conv2':{'weight_bit':8, 'activation_bit':8},
+        'layer4.0.conv1':{'weight_bit':8, 'activation_bit':8},
+        'layer4.0.conv2':{'weight_bit':8, 'activation_bit':8},
+        'layer4.1.conv1':{'weight_bit':8, 'activation_bit':8},
+        'layer4.1.conv2':{'weight_bit':8, 'activation_bit':8},
+        'fc':{'weight_bit':8, 'activation_bit':8},
+    }
```

Comment:
We haven't supported it because not all of ops have been supported in quantization. But we can specify one bit for specific supported op type and all layers of this op will be quantized.